ranjanas20/refactor-SR-Front
# Fix Issue-12: ERROR TypeError: Cannot read property 'map' of undefined

Motivation
 #12

Test

1. Go to http://localhost:4200/
2. Open the developer console in the web browser
3. Navigate to http://localhost:4200/org-location/4/7
 OR search for "organization 4", open it and select a location (Countries and/or states where Organization 4 is involved) 
4. Verify on console , there is no ERROR (TypeError: Cannot read property 'map' of undefined )
